docs/application/dotnet/guides/app-management/widget-control.md

```diff
@@ -0,0 +1,149 @@
+
+# Widget Control
+
+It also provides functions for the following:
+
+*   Sending update requests to the widget applications
+*   Retrieving details of running instance for the same package widget application
+
+
+The main features of the `Tizen.Applications.WidgetControl` class include:
+
+-   [Getting information on widget applications](#getting_information)
+
+    For installed (but not necessarily running) widget applications, you can retrieve widget information with the [Tizen.Applications.WidgetControl](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.WidgetControl.html) class.
+
+
+-   [Listening widget lifecycle events on widget applications](#listening_events)
+
+    For running widget applications, you can listen widget application lifecycle events with the [Tizen.Applications.WidgetControl](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.WidgetControl.html) class.
+
+
+-   [Communicating with running widget instances](#communicating_instances)
+
+    For running widget instances, you can trigger an update event, sending updated data, and retrieving running widget instances detail with the [Tizen.Applications.WidgetControl](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.WidgetControl.html) class.
+
+
+## Prerequisites
+
+To enable your application to use the widget control functionality:
+
+1.  Include the [Tizen.Applications](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.html) namespace in your application to use the methods and properties of the [Tizen.Applications.WidgetControl](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.WidgetControl.html) class
+
+    ```
+    using Tizen.Applications;
+    ```
+
+2.  Request permission by adding the following privilege to the `tizen-manifest.xml` file:
```

Comment:
Please change to:
To get information on widget application, the application has to request permission by adding the following privilege to the  `tizen-manifest.xml` file:

Reply:
Done